grails-app/conf/Config.groovy

```diff
@@ -167,6 +167,7 @@ log4j.main = {
 	warn 'grails.app'
 //    debug 'grails.app'
 //    debug 'liquibase'
+//    debug 'org.bbop.apollo'
```

Comment:
adding this allows debug logging in integration tests. as far as i can tell, didnt output debug logging with just grails.app